content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -0,0 +1,50 @@
+# Amazon SageMaker
+
+## Setup
+
+Many DVC features rely on Git. To work with DVC in Amazon SageMaker, first setup
+your Git repo:
+
+1. [Clone] a repository.
+
+   ![Git clone](/img/sagemaker-git-clone.gif)
+
+2. Launch a terminal or notebook and configure the user name and email:
+
+   ```bash
+   git config --global user.name ...
+   git config --global user.email ...
+   ```
+
+Don't forget to install DVC and any other requirements in your environment!
```

Comment:
```suggestion
3. Don't forget to install DVC and any other requirements in your environment!
```
For a consistent look and feel (since this is a key step for us 😝 ) - maybe also add a screenshot or gif of `!pip install dvc` inside a sagemaker nb